src/vcCamera.cpp

```diff
@@ -464,9 +466,22 @@ void vcCamera_HandleSceneInput(vcState *pProgramState, udDouble3 oscMove, udFloa
   // Allow camera movement when left mouse button is holding down.
   if (!pProgramState->modalOpen)
   {
-    keyboardInput.y += vcHotkey::IsDown(vcB_Forward) - vcHotkey::IsDown(vcB_Backward);
-    keyboardInput.x += vcHotkey::IsDown(vcB_Right) - vcHotkey::IsDown(vcB_Left);
-    keyboardInput.z += vcHotkey::IsDown(vcB_Up) - vcHotkey::IsDown(vcB_Down);
+    bool enableMove = true;
+    ImGuiContext *p = ImGui::GetCurrentContext();
+    if (p && p->ActiveIdWindow)
+    {
+      const char *activeIdName = p->ActiveIdWindow->Name;
+      size_t len = udStrlen(activeIdName);
+      if (len > 0 && udStrstr(activeIdName, len, "###sceneDock") == nullptr) enableMove = false;
```

Comment:
`enableMove = false;` should be on its own line.

Reply:
Fixed. I forgot many times. :(

Reply:
It's all good :)